dotnet/src/webdriver/Internal/Base64UrlEncoder.cs

```diff
@@ -19,8 +19,13 @@
 
 using System;
 
+#nullable enable
+
 namespace OpenQA.Selenium.Internal
 {
+    // When we target .NET 9+, we can use the built-in Base64Url type
```

Comment:
Somewhere in this file there is already a link to a source where we stole it.

So on top of this file I would expect:
```
/*
Based on: https:// github...(link in this file)
Now it is a part of .NET 9 (System.Blah.EncodedUrl type)
/*
```

Reply:
What do you think of the new message?

Reply:
I am OK (in this case) to have a ref from source code to BTS. Please remove a link somewhere in the middle of file.

Reply:
done